FAILED: DispatcherModelSpec

https://jenkins.akka.io:8498/job/akka-nightly-jdk11/207/testReport/akka.actor.dispatch/DispatcherModelSpec/A_Dispatcher_must_process_messages_in_parallel/

```
Error Message

java.lang.AssertionError: assertion failed: expected: Terminated Actor[akka://ActorModelSpec/user/$0p#305110291] but got unexpected message Terminated(Actor[akka://ActorModelSpec/user/$1p#-574121698])

Stacktrace

sbt.ForkMain$ForkError: java.lang.AssertionError: assertion failed: expected: Terminated Actor[akka://ActorModelSpec/user/$0p#305110291] but got unexpected message Terminated(Actor[akka://ActorModelSpec/user/$1p#-574121698])
	at scala.Predef$.assert(Predef.scala:219)
	at akka.testkit.TestKitBase.expectMsgPF(TestKit.scala:437)
	at akka.testkit.TestKitBase.expectMsgPF$(TestKit.scala:433)
	at akka.testkit.TestKit.expectMsgPF(TestKit.scala:923)
	at akka.testkit.TestKitBase.expectTerminated(TestKit.scala:451)
	at akka.testkit.TestKitBase.expectTerminated$(TestKit.scala:450)
	at akka.testkit.TestKit.expectTerminated(TestKit.scala:923)
	at akka.actor.dispatch.DispatcherModelSpec.$anonfun$new$37(ActorModelSpec.scala:595)
```

--- a/akka-actor-tests/src/test/scala/akka/actor/dispatch/ActorModelSpec.scala
+++ b/akka-actor-tests/src/test/scala/akka/actor/dispatch/ActorModelSpec.scala
@@ -574,12 +574,12 @@ class DispatcherModelSpec extends ActorModelSpec(DispatcherModelSpec.config) {
 
   "A " + dispatcherType must {
     "process messages in parallel" in {
-      val probe = TestProbe()
+      val probeA, probeB = TestProbe()
       implicit val dispatcher = interceptedDispatcher()
       val aStart, aStop, bParallel = new CountDownLatch(1)
       val a, b = newTestActor(dispatcher.id)
-      probe.watch(a)
-      probe.watch(b)
+      probeA.watch(a)
+      probeB.watch(b)
 
       a ! Meet(aStart, aStop)
       assertCountDown(aStart, 3.seconds.dilated.toMillis, "Should process first message within 3 seconds")
@@ -592,8 +592,8 @@ class DispatcherModelSpec extends ActorModelSpec(DispatcherModelSpec.config) {
       system.stop(a)
       system.stop(b)
 
-      probe.expectTerminated(a)
-      probe.expectTerminated(b)
+      probeA.expectTerminated(a)
+      probeB.expectTerminated(b)
 
       assertRefDefaultZero(a)(registers = 1, unregisters = 1, msgsReceived = 1, msgsProcessed = 1)
       assertRefDefaultZero(b)(registers = 1, unregisters = 1, msgsReceived = 1, msgsProcessed = 1)

